Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -38,7 +39,8 @@ public function __construct(SymfonyFixturesLoader $fixturesLoader, ?ManagerRegis
             ), E_USER_DEPRECATED);
         }
 
-        parent::__construct($doctrine);
+        // @todo The method_exists call can be removed once the DoctrineBundle dependency has been bumped to at least 1.10
```

Comment:
Can't you bump the dependency and release a new minor instead?

Reply:
That would still leave a broken version for people on older versions of DoctrineBundle, which I'd like to avoid.

Reply:
Fair enough, as long as this diff does get changed when merging up 👍